run_pacta_data_preparation.R

```diff
@@ -1,31 +1,39 @@
+logger::log_threshold(Sys.getenv("LOG_LEVEL", "INFO"))
+logger::log_formatter(logger::formatter_glue)
+
 # necessary packages -----------------------------------------------------------
 
 suppressPackageStartupMessages({
   library(pacta.data.preparation)
   library(pacta.data.scraping)
   library(pacta.scenario.preparation)
-
   library(DBI)
   library(dplyr)
   library(readr)
   library(rlang)
   library(RSQLite)
   library(stringr)
   library(tidyr)
-
-  # used for logging
-  library(rlog)
-  if (interactive()) Sys.setenv("LOG_LEVEL" = "ERROR")
```

Comment:
replace with `library(logger)`?

Reply:
I've made all the logging calls explicit with `logger::log_info()` (and friends).